core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/clusterdefinition/responses/ClusterDefinitionV4Responses.java

```diff
@@ -0,0 +1,20 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.clusterdefinition.responses;
+
+import java.util.Set;
+
+import com.google.common.collect.Sets;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.responses.GeneralCollectionV4Response;
+
+import io.swagger.annotations.ApiModel;
+
+@ApiModel
+public class ClusterDefinitionV4Responses extends GeneralCollectionV4Response<ClusterDefinitionV4Response> {
+
+    public ClusterDefinitionV4Responses(Set<ClusterDefinitionV4Response> responses) {
+        super(responses);
+    }
+
+    public ClusterDefinitionV4Responses() {
+        super(Sets.newHashSet());
```

Comment:
Could you please avoid using `com.google.common.collect.Sets`? I prefer `Collections.emptySet()` because cheaper than construct a new object.

Reply:
My response here disappeared ... or I never submitted it or something. Anyway, I chatted with Richard in other places. I'm concerned this needs to be a mutable set. I've switched it to an ordinary construction of a new HashSet.